fastcrypto/src/lib.rs

```diff
@@ -8,8 +8,6 @@
     rust_2021_compatibility
 )]
 
-pub use signature::Signature as _;
```

Comment:
It's safe to remove this once https://github.com/MystenLabs/sui/pull/9914 is merged